documentation/guides/owner-journey.md

```diff
@@ -71,65 +77,67 @@ gwa status
 
 ## 4. Review Setup
 
-To review what configuration has been setup, log into the [API Services Portal](https://api-gov-bc-ca.test.api.gov.bc.ca), go to the `Namespaces` tab, and select the newly created namespace.
+To review what configuration has been setup, log into the [API Services Portal](https://api-gov-bc-ca.test.api.gov.bc.ca), go to the **Namespaces** tab, and select the newly created Namespace.
 
-From here, you can go to `Activity` to review what configuration has been setup.
+From here, go to **Activity** and select **More details** next to "published gateway configuration" to review what the configuration.
 
 What you have setup:
 
-- Vanity url: `<MYSERVICE>.dev.api.gov.bc.ca`
+- Route to your service, via the vanity url: `<MYSERVICE>.dev.api.gov.bc.ca`
```

Comment:
@ikethecoder this is a little confusing, since when running on the test instance the URL ends up being https://<MYSERVICE>-dev-api-gov-bc-ca.test.api.gov.bc.ca

the behaviour in test.api.gov.bc.ca is to mimic the would-be URL all in the subdomain?

Reply:
```suggestion
- Route to your service, via the vanity URL: `<MYSERVICE>-dev-api-gov-bc-ca.test.api.gov.bc.ca/`
  - If this service were created on the production instance of the API Services Portal, the vanity URL would be `<MYSERVICE>.dev.api.gov.bc.ca`
```
since the tutorial is otherwise framed around working on `test`